internal/runtime/init/init.go

```diff
@@ -127,10 +127,15 @@ func LoadPaths(paths []string,
 		caps = ast.CapabilitiesForThisVersion()
 	}
 
+	var targzs, nonTargzs []string
+	if !asBundle {
+		targzs, nonTargzs = splitByTarGzExt(paths)
```

Comment:
suggestion: I think it'd be helpful to leave a comment here explaining that tar.gz files are automatically loaded as bundles. Regardless of `asBundle` being set.